docs/operations-guide/encrypting-database-details-at-rest.md

```diff
@@ -11,9 +11,11 @@ Your secret key must be at least 16 characters (longer is even better!), and we
 This gives you a cryptographically-secure, randomly-generated 32-character key that will look something like `IYqrSi5QDthvFWe4/WdAxhnra5DZC3RKx3ZSrOJDKsM=`. Set it as an environment variable and
 start Metabase as usual:
 
-    MB_ENCRYPTION_SECRET_KEY='IYqrSi5QDthvFWe4/WdAxhnra5DZC3RKx3ZSrOJDKsM=' java -jar metabase.jar
+    MB_ENCRYPTION_SECRET_KEY="IYqrSi5QDthvFWe4/WdAxhnra5DZC3RKx3ZSrOJDKsM=" java -jar metabase.jar
+
+**Note** Single-quotes (`'`) and double-quotes (`"`) are interpreted differently for environment variable values on some versions of Linux, so when upgrading to a newer version of Linux, then it might be needed to wrap it in double-quotes, if single-quotes were used originally, so the single-quotes are interpreted literally. Ex. `MB_ENCRYPTION_SECRET_KEY='IYq...sM='` would be `MB_ENCRYPTION_SECRET_KEY="'IYq...sM='"`
```

Comment:
```suggestion
**Note** Some versions of Linux interpret single-quotes (`'`) and double-quotes (`"`) differently for environment variable values, so if you upgrade to a newer version of Linux, and the key originally used single-quotes, you might need to wrap the key in double-quotes, so that the single-quotes are interpreted literally. For example, `MB_ENCRYPTION_SECRET_KEY='IYq...sM='` would be `MB_ENCRYPTION_SECRET_KEY="'IYq...sM='"`
```

Reply:
Good catch, Flamber. The single v double quote seems like the kind of thing that could cost users hours trying to figure out.